### ISSUE TITLE
FR: add Area (for use in area card)

### ISSUE THREAD

**Issue by Mariusthvdb** [thomasloven/lovelace-auto-entities#245]
trying to auto list all Areas, I got my hopes up when I saw the area filter and started with:
```
  - type: custom:auto-entities
    card:
      type: grid
      title: Areas
      columns: 2
    card_param: cards
    filter:
      include:
        - area: '*'
          options:
            type: area
```

which unfortunately did not do what I had been looking for... it tries to math any (*) entity to an area...

I then figured maybe try a template filter (which isnt too obvious either, because we need to 'reverse engineer ' the same issue: template an area_id out of the entities:

```
      template: >
        {% set ns = namespace(areas=[])%}
        {% for l in expand(states.light,states.switch)
            if not area_id(l.entity_id) in ns.areas and
            area_id(l.entity_id) is not none %}
          {% set ns.areas = ns.areas + [area_id(l.entity_id)] %}
        {% endfor %}
        {{ns.areas|sort}}
```
got me almost all of my Areas because I have a light or switch in most of them..

However, this doesnt work in auto-entities either.
Bottom line for now is I believe we can not use Auto-entities to list Areas at all?

Hence this FR:

Please add an Area option to auto-entities so we can do something like my top config?

thanks for considering

**Comment by thomasloven** [comment]
Areas are not entities.
